subsys/bluetooth/mesh/rpl.c

```diff
@@ -27,63 +28,43 @@
 #include "adv.h"
 #include "net.h"
 #include "rpl.h"
-#include "settings.h"
-
-/* Replay Protection List information for persistent storage. */
-struct rpl_val {
-	uint32_t seq:24,
-	      old_iv:1;
-};
+#include "rpl_storage.h"
 
+/* Covers no persistent storage usecase */
+#ifdef CONFIG_BT_MESH_RPL_STORAGE_MODE_NONE
```

Comment:
I guess replay_list in any case should stay in rpl_storage.c together with other settings code.

Reply:
Ok, so I'll create rpl_storage.c and link it in if there is no persistent storage then